### ISSUE TITLE
Designing&Implementing the home page of our app

### ISSUE THREAD

**Issue by berkaydoner** [bounswe/2021SpringGroup2#143]
We should design and implement the home page of our application. This home page will be the first page the users will see, therefore it should be user-friendly and practical. It should have many components, for now I can think of the following:
- [ ] Header, where links to other pages are given
- [ ] Sidebar, where user can interact with their events, equipment, applications and so on
- [ ] Search bar, where users can search and filter events/equipment
- [ ] Maybe a list of events that are similar to the previous events of the user

Last two will be functional only after we implement search and recommendation systems. However, it might be nice to display these areas with dummy data for now so that we can have a complete design.

**Comment by CenekSanzak** [comment]
I created the card component for events on the homepage. I also created a controller for fetching and processing the sports data.

**Comment by berkaydoner** [comment]
I have prepared the initial design of the home page. I did not implement the header&sidebar, Şefika will implement them and then I will add their components to the home page.
I decided to list some events in the home page, as I thought earlier. With Doğukan, we decided that we can use card components to represent such events. We also planned to list some sports genre cards, which can enhance the user experience. As Doğukan is more experienced in frontend than me, he also initialized the card structures and helped me to understand the components better.
I used 1x3 grid structures for sports and events in the homepage, there are 12 cards and user can navigate between these cards with backward and forward buttons. In sports cards, title and the descriptions are given together with the image of the sport. I also used these images to represent the event types in event cards, as can be seen below in the figure. For all sports information, I used [Decathlon API](https://developers.decathlon.com/products/sports/docs), which can slow down the process. We might store the related sports data in our database later to speed up our web page.

![FireShot Capture 007 -  - localhost](https://user-images.githubusercontent.com/52797716/141684874-1f098998-e5c7-4332-84a3-308aebc67dd3.png)

**Comment by berkaydoner** [comment]
I fixed the event card layout. Now, icons and related information are stacked vertically and the description of the event is at the center. I also aligned the "Discover Sports" and "Browse Events" titles and improved their font types, sizes, color and positions.


![FireShot Capture 008 -  - localhost](https://user-images.githubusercontent.com/52797716/141699859-b5682775-7f4a-42d4-a64d-7ca5ca87e7b3.png)

**Comment by berkaydoner** [comment]
For now, the page is not connected to the backend since get events functionality of backend was not documented yet, I will add it later on in another issue. We also decided that we will further improve the design of the homepage together as the frontend team. We can open separate issues then, I am closing this issue.
